bee-inx/examples/milestone.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use bee_inx::{client, Error};
+use futures::StreamExt;
+
+const INX_ADDRESS: &str = "http://localhost:9029";
+
+#[tokio::main]
+async fn main() -> Result<(), Error> {
+    let mut inx = client::Inx::connect(INX_ADDRESS.into()).await?;
+    let mut milestone_stream = inx.listen_to_confirmed_milestones((..).into()).await?;
+
+    // Listen to the milestones from the node.
+    while let Some(milestone) = milestone_stream.next().await {
+        let milestone_index = milestone?.milestone_info.milestone_index;
+        println!("Fetch cone of milestone {milestone_index}");
+
+        // Listen to messages in the past cone of a milestone.
+        let mut cone_stream = inx.read_milestone_cone(milestone_index.into()).await?;
+
+        // Keep track of the number of blocks.
+        let mut count = 0usize;
+
+        while let Some(Ok(block_metadata)) = cone_stream.next().await {
+            println!(
+                "Block {}: {:#?}",
+                block_metadata.metadata.block_id, block_metadata.block
+            );
+            count += 1;
+        }
+
+        println!("Milestone {:?} contained {count} blocks", milestone_index);
```

Comment:
I know it's just an example, but I think this should be spawned in a task.

Reply:
Also...it's probably enough to just print the `BlockId` for each block in the the cone. Otherwise the console will just be spammed with data that doesn't really add much to the example.

Reply:
Why do you think this should be done in a task? Also, I've changed the output.

Reply:
Just to make it faster on multi-core machines I guess, and essentially make the `milestone` processing more independent from how long the cone takes to be processed.

Reply:
But then the output would not be in order anymore, because multiple tasks could work on this concurrently? I don't know if it's worth the additional complexity.

Reply:
Yea, the output would potentially not be in order anymore. I didn't know that this is a requirement for this example.

Reply:
If you want, you can always provide a more elaborate example! :trollface: